src/AddTodoForm.jsx

```diff
@@ -0,0 +1,13 @@
+const AddTodoForm = () => {
+    return (
+        <>
+            <form action="">
```

Comment:
In one of the next few lessons, we're going to handle the event of submitting this form! This uses a different attribute on the form element, so let's remove this `action` attribute for now.